docs/eventing/debugging/README.md

```diff
@@ -10,8 +10,8 @@ know roughly how things fit together.
 ## Version
 
 This document works with
-[Eventing 0.3](https://github.com/knative/eventing/releases/tag/v0.3.0) and
-[Eventing Sources 0.3](https://github.com/knative/eventing-sources/releases/tag/v0.3.0).
+[Eventing 0.6](https://github.com/knative/eventing/releases/tag/v0.6.0) and
```

Comment:
Any chance we can use 0.7.0 or 0.7.1 instead?

I realize that this is only because the review has been so slow, so feel free to say we can do it in a follow up.